### DISCUSSION TITLE
Raspberry Pi 5.5inch 1440x2560 LCD

### DISCUSSION THREAD

**Discussion by Alex256256** [rern/rAudio#1485] [Feature Requests]
Hey guys, a week later, I'm trying to flip the portrait display to landscape mode. Google searches yield no results because we do not use xrandr and arandr. how to turn a 1440x2560pix display into landscape mode (config.txt does not help, resolution is 1440x1440 no more). I believe that the browser needs to be launched with the command to rotate the display, but I don’t know how to do this. Please, help!
![IMG_20231106_150115](https://github.com/rern/rAudio/assets/14962189/4ca05d02-831a-4a4b-b419-5778da4ef19e)
![IMG_20231106_150152](https://github.com/rern/rAudio/assets/14962189/f0069393-f978-497d-90b4-eaccda50b5dc)
[config.txt](https://github.com/rern/rAudio/files/13265193/config.txt)

**Comment by rern** [comment]
Hi,

Please try: Features >  Browser on RPi - Rotation

**Reply by Alex256256** [reply]
Hi, where can I find this feature? In the rAudio menu

**Reply by Alex256256** [reply]
![IMG_20231106_180837](https://github.com/rern/rAudio/assets/14962189/4fd0a16f-7e45-4f0d-98c0-e323958b5561)
Cool man! it really works) I want to know if there is a user manual?)
 and here’s another question: is it worth connecting FKMS? (dtoverlay=vc4-fkms-v3d (config.txt)) what graphics settings work best for audio?
